dist/FxAccountsPairingChannel.babel.umd.js

```diff
@@ -7897,7 +7904,9 @@ if (
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 // A wrapper that combines a WebSocket to the channelserver
 // with some client-side encryption for securing the channel.
-// We'll improve the encryption before initial release...
```

Comment:
We actually did "improve the encryption" quite substantially before initial release 😅